tests/subsys/llext/simple/src/test_llext_simple.c

```diff
@@ -415,7 +415,7 @@ ZTEST(llext, test_find_section)
 	res = llext_load(loader, "find_section", &ext, &ldr_parm);
 	zassert_ok(res, "load should succeed");
 
-	section_ofs = llext_find_section(loader, ".data");
+	section_ofs = llext_section_offset(loader, ext, ".data");
```

Comment:
Since `find_section` is not deprecated, both should be tested here.

Reply:
we plan to deprecate it - once we've updated SOF to not call it to avoid twister failures